packages/commerce-sdk-react/src/hooks/useCommerceApi.ts

```diff
@@ -5,48 +5,17 @@
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 
-import {
-    ShopperBaskets,
-    ShopperContexts,
-    ShopperCustomers,
-    ShopperLogin,
-    ShopperOrders,
-    ShopperProducts,
-    ShopperPromotions,
-    ShopperDiscoverySearch,
-    ShopperGiftCertificates,
-    ShopperSearch
-} from 'commerce-sdk-isomorphic'
-
+import React from 'react'
+import {CommerceApiContext} from '../provider'
 import {ApiClients} from './types'
 
-const config = {
-    proxy: 'https://localhost:3000',
-    headers: {},
-    parameters: {
-        clientId: '<your-client-id>',
-        organizationId: '<your-org-id>',
-        shortCode: '<your-short-code>',
-        siteId: '<your-site-id>'
-    },
-    throwOnBadResponse: true
-}
-
+/**
+ * Access a set of Commerce API clients
```

Comment:
```suggestion
 * Access a set of initialized Commerce API clients, the clients are initialized using the configurations passed to the CommerceApiProvider.
```

I think the key point here is that, the hook returns the **initialized** api clients based on the config you pass to the provider.